understanding/understanding-template.html

```diff
@@ -1,14 +1,14 @@
 <!DOCTYPE html>
 <html xmlns="http://www.w3.org/1999/xhtml">
 	<head>
-		<title>WCAG 2.0 Understanding Page</title>
+		<title>WCAG 2 Understanding Page</title>
 		<link rel="stylesheet" type="text/css" href="sources.css"/>
 	</head>
 	<body>
 		<h1>Understanding SC #.#.#</h1>
 		<section id="intent">
 			<h2>Intent</h2>
-			<p class="instructions">This section contains the main explanatory content of the Understanding. It explains why the Guideline or Success Criterion exists and, at a high level, how to meet it.</p>
+			<p class="instructions">This section contains the main explanatory content of the Understanding. It explains why the guideline or success criterion exists and, at a high level, how to meet it.</p>
```

Comment:
I have a mild concern with removing the initial cap on "Guideline" here, simply because it specifically refers to the dozen or so Guidelines, but could be misread to refer to the entire document.
There is no way anyone could misunderstand "success criterion", but I think the capital G here helps understand what is being referred to.
```suggestion
			<p class="instructions">This section contains the main explanatory content of the Understanding. It explains why the Guideline or success criterion exists and, at a high level, how to meet it.</p>
```